internal/command/apps/apps.go

```diff
@@ -26,13 +33,38 @@ The LIST command will list all currently registered applications.
 		newList(),
 		newCreate(),
 		newDestroy(),
+		newRestart(),
 		newMove(),
 		newResume(),
 		newSuspend(),
-		newRestart(),
 		NewOpen(),
 		NewReleases(),
 	)
 
 	return apps
 }
+
+// BuildContext is a helper that builds out commonly required context requirements
+func BuildContext(ctx context.Context, app *api.AppCompact) (context.Context, error) {
```

Comment:
Is this mainly to get the flaps client into context?

Reply:
Both the flaps client and dialer.